src/App.tsx

```diff
@@ -1,25 +1,131 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: getUser(todo.userId),
+  };
+});
+
+export const App: React.FC = () => {
+  const [initialTodo, setInitialTodo] = useState<Todo[]>(todos);
+
+  const [userId, setUserId] = useState(0);
+  const [selectErrorMessage, setSelectErrorMessage] = useState(false);
```

Comment:
Bad naming here too. It's not clear that the value is boolean. Change the name or use empty string as the value